website/docs/reference/resource-configs/delimiter.md

```diff
@@ -0,0 +1,102 @@
+---
+resource_types: [seeds]
+datatype: <string>
+default_value: ","
+---
+
+## Definition
+
+An optional seed configuration used to customize how you separate values in a [seed](/docs/build/seeds) with the one-character string you provide.
```

Comment:
Might just be me but this reads like it's changing the way the values are separated in the seed file, but if I understand this correctly, what it's doing is telling dbt how the values are separated (comma for CSV, newlines for TSV, and so on)

If I don't have that backwards, maybe something like this:
```suggestion
An optional seed configuration used to define how values are separated in a [seed](/docs/build/seeds) with the one-character string you provide.
```